docs/inputs.md

```diff
@@ -1707,6 +1731,159 @@ regional_pass: 10.90
+    </tr>
+    <tr>
+        <td>Commodity_Group</td>
+        <td>Determines the commodity group (15 categories) to which each of the 43 commodities from the FAF data belongs.
+</td>
+    </tr>
+    <tr>
+        <td>EMP_Calc</td>
+        <td>Calculate the share of each of the 3 digits NAICS employee categories within the 19 categories SANDAG ABM employee for each of the 5 FAZs in San Diego County</td>
+    </tr>
+    <tr>
+        <td>EMP_Converter</td>
+        <td>Provides a table that correlates SANDAG model employee categories with corresponding NAICS employee categories</td>
+    </tr>
+    <tr>
+        <td>CG_Emp_P</td>
+        <td>Establishes the relationship between each Commodity group and the NAICS employee category for the production end.</td>
+    </tr>
+    <tr>
+        <td>CG_Emp_A</td>
+        <td>Establishes the relationship between each commodity group and the NAICS employee category for the attraction end.</td>
+    </tr>
+    <tr>
+        <td>Annual_Factor</td>
+        <td>Number of business days in a year</td>
+    </tr>
+    <tr>
+    <td>Truck_Dist</td>
+    <td>Provides the percent distribution of truck type based on OD distance. </td>
+    </tr>
+    <tr>
+    <td>Payload</td>
+    <td>Average pounds of load that each truck type can carry based on commodity groups. </td>
+    </tr>
+    <tr>
+    <td>Time_of_Day</td>
+    <td>Provides distribution of trucks throughout the day </td>
+    </tr>
+    <tr>
+    <td>External_Count</td>
+    <td>The Inbound and outbound truck counts by type at each of the 12 SANDAG gateways. For base year, this is the daily truck counts at the gateways.
+ </td>
+    </tr>
+    <tr>
+    <td>SRA_Dist</td>
+    <td>The overall distribution of trucks when they cross each of the gateway from/to each of the 63 SRAs. For base year this information is calculated from truck GPS data </td>
+    </tr>
+    <tr>
+    <td>SRA_TAZ</td>
+    <td>SRA-TAZ Mapping</td>
+    </tr>
+</table>
+
+<a id="heavy_truck_model_baseAndFutureYears"></a>
+
+### Heavy Truck Model Inputs
+#### `FAF5_BaseAndFutureYears_Oct27_2023.csv`
+
+<table>
+    <tr>
+        <th>Column Name</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td>dms_orig</td>
+        <td>FAF region or state where a freight movement begins the domestic portion of shipment. 
+For imports, this is the US entry region where an import enters United States.</td>
+    </tr>
+    <tr>
+        <td>dms_dest</td>
+        <td>FAF region or state where a freight movement ends the domestic portion of shipment. 
+For exports, this is the US exit region where an export leaves United States.</td>
+    </tr>
+    <tr>
+        <td>Mode</td>
+        <td>Mode used between domestic origins and destinations</td>
+    </tr>
+    <tr>
+        <td>Commodity</td>
+        <td>2-digit level of the Standard Classification of Transported Goods (SCTG)
+</td>
```

Comment:
Please double check from line #1833 to line#1847 to make sure format looks good.